src/main/webapp/resources/js/pages/projects/samples-metadata-import/components/SampleMetadataImportSelectFile.tsx

```diff
@@ -120,20 +130,42 @@ export function SampleMetadataImportSelectFile(): JSX.Element {
         });
         return false;
       }
-      return true;
     },
   };
 
   return (
     <SampleMetadataImportWizard current={0} status={status}>
       <Spin spinning={loading}>
-        {!valid && (
+        {(showEmptyHeaderError || showDuplicateHeaderError) && (
           <ErrorAlert
             style={{ marginBottom: SPACE_XS }}
             message={i18n("SampleMetadataImportSelectFile.alert.valid.title")}
-            description={i18n(
-              "SampleMetadataImportSelectFile.alert.valid.description"
-            )}
+            description={
+              <>
+                {i18n(
+                  "SampleMetadataImportSelectFile.alert.valid.description.preface"
```

Comment:
Can we bold the preface and the postface? You can use the Typography component for this

Reply:
Please see a54ec79fd261822b9a8c159c45f7c4db4a1fb00e.